dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -1079,8 +1079,8 @@ export const p2pDataModel = [
           {
             name: "excessNitrogen",
             label: "Excess nitrogen",
-            description: "Excess nitrogen in kg/ha",
-            unit: "",
+            description: "Excess nitrogen",
+            unit: "kg / ha",
```

Comment:
Again, Unit now no longer visible in upload page. Only acceptable if this will be resolved as part of DALA-2373.